src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -501,11 +501,12 @@ static uct_iface_ops_t uct_rc_verbs_iface_tl_ops = {
 static uct_rc_iface_ops_t uct_rc_verbs_iface_ops = {
     .super = {
         .super = {
-            .iface_estimate_perf = uct_rc_iface_estimate_perf,
-            .iface_vfs_refresh   = uct_rc_iface_vfs_refresh,
-            .ep_query            = (uct_ep_query_func_t)ucs_empty_function_return_unsupported,
-            .ep_invalidate       = (uct_ep_invalidate_func_t)ucs_empty_function_return_unsupported,
-            .ep_connect_to_ep_v2 = uct_rc_verbs_ep_connect_to_ep_v2
+            .iface_estimate_perf   = uct_rc_iface_estimate_perf,
+            .iface_vfs_refresh     = uct_rc_iface_vfs_refresh,
+            .ep_query              = (uct_ep_query_func_t)ucs_empty_function_return_unsupported,
+            .ep_invalidate         = (uct_ep_invalidate_func_t)ucs_empty_function_return_unsupported,
+            .ep_connect_to_ep_v2   = uct_rc_verbs_ep_connect_to_ep_v2,
+            .iface_is_reachable_v2 = (uct_iface_is_reachable_v2_func_t)uct_ib_iface_is_reachable_v2
```

Comment:
can we remove the type casting? (in multiple places)

Reply:
Looks we can't. They have difference return type, int vs status_t.